core/src/com/unciv/models/ruleset/unit/BaseUnit.kt

```diff
@@ -667,7 +667,7 @@ class BaseUnit : RulesetObject(), INonPerpetualConstruction {
                 unique.isOfType(UniqueType.StrengthNearCapital) && unique.params[0].toInt() > 0 ->
                     power *= (unique.params[0].toInt() / 4f).toPercent()  // Bonus decreasing with distance from capital - not worth much most of the map???
 
-                unique.placeholderText == "May Paradrop up to [] tiles from inside friendly territory" // Paradrop - 25% bonus
+                unique.placeholderText == UniqueType.MayParadrop.placeholderText // Paradrop - 25% bonus
```

Comment:
`unique.isOfType(UniqueType.MayParadrop)`?
Or, even better, changing this entire when to query the type of a unique (`when (unique.type) {`)?

Reply:
After all, that's the point of the exercise.

Reply:
Initially, I thought about the same, however, `isOfType` does even more complex magic under the hood `UniqueType.values().firstOrNull { it.placeholderText == placeholderText }` which ends up with comparing the `placeholderText`, anyway. Thus I decided to keep it more obvious and simple.

Reply:
Fair to say, I like the idea to have `when (unique.type) {` , probably, we can even get rid of `placeholderText` property at some point. I don't like the fact we store the same data in 2 places: in `placeholderText` and in `type.placeholderText` de facto.

Reply:
`isOfType` using placeholderText is an interim kludge until all base game uniques are typed - or so I always thought? So using it while still biting your nails at its inefficiency is still a good thing :tm: 

`text` vs. `placeholderText` is not so easy - they're not the same data, and deriving the latter from the former on the fly is possible at a RegEx expense.